src/ngraph/runtime/cpu/cpu_backend.cpp

```diff
@@ -145,3 +145,13 @@ vector<runtime::PerformanceCounter>
     }
     return rc;
 }
+
+bool runtime::cpu::CPU_Backend::is_supported_property(const Property prop) const
+{
+    if (prop == Property::memory_attach)
+    {
+        return true;
```

Comment:
why is this true? what about the CPU backend makes this property supported. Is there a way to test that a backend does indeed support the property.

Reply:
@jbobba As far as I see the CPU has memory attach ability as already used in nGraph-TF https://github.com/NervanaSystems/ngraph-tf/blob/master/src/ngraph_encapsulate_op.cc#L316